src/bridge/bridge-unix-sockets.cpp

```diff
@@ -152,7 +156,12 @@ bool sendBridgeMessage(messages::ProtobufMessage& message, SlimeVRDriver::VRDriv
 BridgeStatus runBridgeFrame(SlimeVRDriver::VRDriver& driver) {
     try {
         if (!client.IsOpen()) {
-            client.Open(SOCKET_PATH);
+            if(const char* ptr = std::getenv("XDG_RUNTIME_DIR")) {
+                const fs::path xdg_runtime = ptr;
+                client.Open((xdg_runtime / SOCKET_NAME).native());
+            } else {
+                client.Open((fs::path(TMP_DIR) / SOCKET_NAME).native());
+            }
```

Comment:
Code looks fine to me, though it's worth noting that since environment variables aren't going to change at runtime, we could do this once when the driver starts instead of re-computing it every time we try to open the socket.

that said it's only going to impact people who don't have slimevr server running (hi that's me), and only minimally at that.